static/app/components/superuserAccessForm.tsx

```diff
@@ -127,6 +153,7 @@ class SuperuserAccessForm extends Component<Props, State> {
       error: true,
       errorType,
       showAccessForms: true,
+      isLoading: false,
```

Comment:
Handling means that we've already sent a request with or request the U2F tap. In either case, we should've already loaded the authenticators